### PR TITLE
Adding changes for modular rewrite

### PR DESCRIPTION
**Pull request by bhparijat**

### CODE REVIEW

File: sample/app/package.json [hunk 1]
```diff
@@ -12,11 +12,11 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "express": "^4.16.3",
-    "handlebars": "^4.7.7",
+    "handlebars": "^4.7.6",
```
Comment (prameshj) [inline sample/app/package.json:15]: Why did we bump this down?

File: sample/authui-react/package.json [hunk 2]
```diff
@@ -13,9 +13,9 @@
     "@types/react-router": "^5.1.8",
     "@types/react-router-dom": "^5.1.5",
     "bootstrap": "^4.5.0",
-    "firebase": "^9.1.3",
+    "firebase": "^9.8.3",
     "firebaseui": "^6.0.0",
-    "gcip-iap": "^1.0.1",
+    "gcip-iap": "file:../..",
```
Comment (prameshj) [inline sample/authui-react/package.json:18]: Change to the latest version 2.0.0

File: package.json [hunk 3]
```diff
@@ -8,6 +8,24 @@
   "engines": {
     "node": ">=8.0.0"
   },
+  "scripts": {
```
Comment (prameshj) [inline package.json:11]: change version to 2.0.0 in line 3.

File: sample/authui/package.json [hunk 4]
```diff
@@ -40,9 +40,9 @@
     "@types/node": "^14.0.24",
     "bootstrap": "4.5.0",
     "codelyzer": "^6.0.0",
-    "firebase": "^9.1.3",
+    "firebase": "^9.8.3",
     "firebaseui": "^6.0.0",
-    "gcip-iap": "^1.0.1",
+    "gcip-iap": "file:../..",
```
Comment (prameshj) [inline sample/authui/package.json:45]: let's restore to ^1.0.1

File: package.json [hunk 5]
```diff
@@ -8,6 +8,24 @@
...
+    "build": "gulp build",
```
Comment (prameshj) [inline package.json:12]: Let's remove this section that was added.

**Review by prameshj — APPROVED** [review]

File: sample/app/package.json (hunk 1 above)
Comment (bhparijat) [reply sample/app/package.json:15]: This must have been from audit fix.. I did not do it on my own

File: sample/authui-react/package.json (hunk 2 above)
Comment (bhparijat) [reply sample/authui-react/package.json:18]: Made it ^1.0.1 here and in angular folder

File: sample/authui/package.json (hunk 4 above)
Comment (bhparijat) [reply sample/authui/package.json:45]: Made it ^1.0.1 here and in react folder

**Pull request merged**